contents/spotlight/startup-bugprove.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "How Bugprove uses influencer marketing to grow word of mouth"
+date: 2023-08-31
+author: ["joe-martin"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Startups
+---
+
+> Each month, we speak to a team enrolled in [PostHog for Startups](/startups) about the way they work and the challenges they face. This month we spoke to Bugprove about their approach to developer and influencer marketing. 
+
+[Bugprove](https://bugprove.com/) is a firmware analysis platform for IoT devices which helps security researchers and manufacturers meet their compliance needs. Since 2022 it’s helped teams scan for vulnerabilities, fix them with AI, and monitor emerging threats. 
```

Comment:
Our product launched in 2023 only. So we started working on it in 2022, but we only started helping people from February this year :)